src/pages/GamePage/components/AttributesWrapper.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Attribute from './Attribute';
+
+export default function AttributesWrapper({ stat, side, attributesNames }) {
+  const arr = side === 'right' ? [0, 1, 0] : [1, 0, 1];
```

Comment:
this logic could be wrapped in a function, also use better names for the variables (Clean code)